### ISSUE TITLE
[Bug]: Fix Integration tests post #304

### ISSUE THREAD

**Issue by BradyPlanden** [pybop-team/PyBOP#320]
### Python Version

N/A

### Describe the bug

#304 has reduced the test suite completion time, but has resulted in a higher failure rate of the integration tests. A fix is available in #319 and will be duplicated into this branch to expedite the fix.

### Steps to reproduce the behaviour

See: https://github.com/pybop-team/PyBOP/actions/runs/9019436453/job/24782313287

**Comment by NicolaCourtier** [comment]
I also saw this when I merged develop into #236, changing the scale in line 15 in tests/integration/test_parameterisations.py seems to help. I reduced it from 0.05 to 0.03 in that PR.

**Comment by BradyPlanden** [comment]
Thanks @NicolaCourtier. I've opened a PR for this fix now. I was able to fix this issue without changing the range of potential parameter values. This was achieveable without extending the synthetic prediction, but the extension should help with identifiability of the fitting parameters in this case. 

It should be ready for review soon, I want to run the scheduled workflow on the branch for confirmation. On initial testing, this change appears to allow for tighter assertions on the parameterisations test, which would be great.
